README.md

```diff
@@ -69,9 +69,18 @@ To set up a development environment for the CLI
 
 1. Install go v1.17
 2. Install [buf](https://github.com/bufbuild/buf)
-3. make or make unit-tests to ensure everything built correctly.
+3. Run `make all` to install dependencies and build binaries and assets
+4. Start a `kind` cluster like so: `KIND_CLUSTER_NAME=<some name> ./tools/kind-with-registry.sh`
+5. Run `./bin/gitops install --config-repo=<repo url>`
+6. Start the in-cluster API replacement job (powered by [http://tilt.dev](tilt.dev)) with `make cluster-dev`
```

Comment:
so this is a bit awkward here right. Because tilt up will also run a `make bin` overwriting whatever you built with make all anyways. What if we would just run `make all` in tilt up?

Reply:
I think running `make bin` again is fine, as we need to override the `GOOS` and `GOARCH` anyway. Marking as resolved.